hadoop-ozone/dist/src/main/smoketest/security/ozone-secure.robot

```diff
@@ -17,138 +17,42 @@
 Documentation       Smoke test to start cluster with docker-compose environments.
 Library             OperatingSystem
 Library             String
+Library             BuiltIn
 Resource            ../commonlib.robot
 
 *** Variables ***
-${ENDPOINT_URL}       http://s3g:9878
+${ENDPOINT_URL}    http://s3g:9878
 
 *** Keywords ***
-Install aws cli s3 centos
-    Execute                    sudo yum install -y awscli
-Install aws cli s3 debian
-    Execute                    sudo apt-get install -y awscli
-
-Install aws cli
-    ${rc}              ${output} =                 Run And Return Rc And Output           which apt-get
-    Run Keyword if     '${rc}' == '0'              Install aws cli s3 debian
-    ${rc}              ${output} =                 Run And Return Rc And Output           yum --help
-    Run Keyword if     '${rc}' == '0'              Install aws cli s3 centos
-
-Setup credentials
-    ${hostname}=        Execute                    hostname
-    Execute             kinit -k testuser/${hostname}@EXAMPLE.COM -t /etc/security/keytabs/testuser.keytab
-    ${result} =         Execute                    ozone s3 getsecret
-    ${accessKey} =      Get Regexp Matches         ${result}     (?<=awsAccessKey=).*
-    ${secret} =         Get Regexp Matches	       ${result}     (?<=awsSecret=).*
-                        Execute                    aws configure set default.s3.signature_version s3v4
-                        Execute                    aws configure set aws_access_key_id ${accessKey[0]}
-                        Execute                    aws configure set aws_secret_access_key ${secret[0]}
-                        Execute                    aws configure set region us-west-1
+Setup volume names
+    ${random}            Generate Random String  2   [NUMBERS]
+    Set Suite Variable   ${volume1}            fstest${random}
+    Set Suite Variable   ${volume2}            fstest2${random}
 
 *** Test Cases ***
 Create volume and bucket
-    ${rc}              ${output} =                 Run And Return Rc And Output              ozone sh volume create o3://om/fstest --user bilbo --quota 100TB --root
-                        Should contain       ${output}       Client cannot authenticate via
+    Execute             kdestroy
```

Comment:
NIT: can be more clean to use two different test case:

1. Create volume bucket with wrong credentials
2. Create volume bucket

Reply:
done